### PR TITLE
dotCMS/core#18310 Move link to info dialog

### PR DESCRIPTION
**Pull request by fmontes**

### CODE REVIEW

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js [hunk 1]
```diff
@@ -140,284 +148,264 @@ dojo.declare("dotcms.dijit.form.FileAjaxUploader", [dijit._Widget, dijit._Templa
...
-            &&  dijit.byId("sortOrder")
-            &&  dojo.byId("metaData_field")
-        );
-
+    _isFileAsset: function () {
+        return dijit.byId("fileName") && dijit.byId("title") && dijit.byId("sortOrder") && dojo.byId("metaData_field");
     },
 
+    _postUploadCallback: function (fileData) {
+        var fieldRelatedData = {
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js:309]: Codacy Issue found: [Avoid trailing commas in object or array literals](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5396940)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js [hunk 2]
```diff
@@ -140,284 +148,264 @@ dojo.declare("dotcms.dijit.form.FileAjaxUploader", [dijit._Widget, dijit._Templa
...
             }
 
             var thumbNailImg = document.createElement("img");
             var thumbnailImage;
 
-            if(!fieldRelatedData.fileName.toLowerCase().endsWith('.svg')) {
-                thumbnailImage = "/contentAsset/image/"+fileData.id+ "/" + this.id +"/?filter=Thumbnail&thumbnail_w=300&thumbnail_h=300";
-            }else{
+            if (!fieldRelatedData.fileName.toLowerCase().endsWith(".svg")) {
+                thumbnailImage =
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js:374]: Codacy Issue found: [Avoid using global variables](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5396940)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js [hunk 3]
```diff
@@ -140,284 +148,264 @@ dojo.declare("dotcms.dijit.form.FileAjaxUploader", [dijit._Widget, dijit._Templa
...
-
         var file = this.fileInputField.files;
-        if(file.length!=1){
-            console.log("upload 1 file only - you have " + file.length)
+        if (file.length != 1) {
+            console.log("upload 1 file only - you have " + file.length);
             return;
         }
-        file=file[0];
+        file = file[0];
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js:173]: Codacy Issue found: [Avoid using global variables](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5396940)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.jsp [hunk 4]
```diff
@@ -1,29 +1,51 @@
...
 <div id="${id}" class="fileAjaxUploader">
+    <input name="${name}" id="${name}ValueField" dojoAttachPoint="fileNameField" type="hidden" value="${fileName}" />
+    <div dojoAttachPoint="fileUploadForm" class="fileAjaxUploaderForm">
+        <input
+            name="${name}FileUpload"
+            type="file"
+            accept="${accept}"
+            dojoAttachEvent="onchange:_doFileUpload"
+            dojoAttachPoint="fileInputField"
+            class="form-text"
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.jsp:11]: Codacy Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5396940)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.jsp [hunk 5]
```diff
@@ -1,29 +1,51 @@
...
+            <button dojoType="dijit.form.Button" dojoAttachEvent="onClick:_info" iconClass="infoIcon">
+                <%= LanguageUtil.get(pageContext, "Info")%>
+            </button>
+        </div>
+        <div dojoAttachPoint="fileUploadRemoveButton" style="display: none;">
+            <button
+                dojoType="dijit.form.Button"
+                dojoAttachEvent="onClick:_remove"
+                iconClass="deleteIcon"
+                class="dijitButtonDanger"
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.jsp:37]: Codacy Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5396940)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js [hunk 6]
```diff
@@ -140,284 +148,264 @@ dojo.declare("dotcms.dijit.form.FileAjaxUploader", [dijit._Widget, dijit._Templa
             </div>';
 
         let warningInfo = {};
-        warningInfo['newAssetName'] = newAssetName;
-        warningInfo['oldAssetName'] = oldAssetName;
+        warningInfo["newAssetName"] = newAssetName;
+        warningInfo["oldAssetName"] = oldAssetName;
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js:152]: Codacy Issue found: [Avoid using global variables](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5396940)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js [hunk 7]
```diff
@@ -140,284 +148,264 @@ dojo.declare("dotcms.dijit.form.FileAjaxUploader", [dijit._Widget, dijit._Templa
...
         let message = dojo.replace(this.replaceAssetNameWarning, warningInfo);
 
         let fileInfo = {};
-        fileInfo['newAssetName'] = newAssetName;
-        fileInfo['oldAssetName'] = oldAssetName;
-        fileInfo['message'] = message;
+        fileInfo["newAssetName"] = newAssetName;
+        fileInfo["oldAssetName"] = oldAssetName;
+        fileInfo["message"] = message;
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js:158]: Codacy Issue found: [Avoid using global variables](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5396940)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js (hunk 1 above)
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js:309]: Codacy Issue found: [Avoid using global variables](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5396940)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js [hunk 8]
```diff
@@ -140,284 +148,264 @@ dojo.declare("dotcms.dijit.form.FileAjaxUploader", [dijit._Widget, dijit._Templa
...
         }
 
-		this.onRemove(this);
-
+        this.onRemove(this);
     },
 
     _info: function () {
-
+        const host = location.protocol + "//" + location.host;
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js:277]: Codacy Issue found: [Avoid using global variables](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5396940)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.jsp [hunk 9]
```diff
@@ -1,29 +1,51 @@
...
+        />
+    </div>
+    <div dojoAttachPoint="fileUploadStatus" class="fileAjaxUploaderStatus" style="display: none;">
+        <div dojoType="dijit.ProgressBar" dojoAttachPoint="progressBar"></div>
+        <div class="fileAjaxUploaderStatusMsg"><%= LanguageUtil.get(pageContext, "Uploading")%>...</div>
+    </div>
+    <div
+        name="${name}FileName"
+        dojoAttachPoint="fileNameDisplayField"
+        class="fileAjaxUploaderFileName"
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.jsp:22]: Codacy Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5396940)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js [hunk 10]
```diff
@@ -47,78 +47,86 @@ dojo.require("dijit.form.Button");
...
     },
 
-    _fileInfoTemplate: function () {
-        return this.fileNameVisible ? '<div>\
-        <table class="listingTable">\
-            <tr class="alternate_1">\
+    
+
+    _fileInfoTemplate: function ({resourceLink, versionPath, idPath}) {
+        const resourceLinkRow = resourceLink && "<tr><td><b>{resourceLinkLabel}</b></td><td><a target=\"_blank\" href=\"{resourceLink}\">{resourceLink}</a></td></tr>";
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js:103]: Codacy Issue found: [Avoid using global variables](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5396940)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.jsp [hunk 11]
```diff
@@ -1,29 +1,51 @@
...
+            </button>
+        </div>
+    </div>
+    <input type="hidden" id="maxSizeFileLimit" value="" />
+    <div
+        id="${id}-Dialog"
+        dojoType="dijit.Dialog"
+        dojoAttachPoint="fileInfoDialog"
+        style="width: 500px;"
+        class="noDijitDialogTitleBar"
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.jsp:49]: Codacy Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5396940)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js [hunk 12]
```diff
@@ -140,284 +148,264 @@ dojo.declare("dotcms.dijit.form.FileAjaxUploader", [dijit._Widget, dijit._Templa
             </div>';
 
         let warningInfo = {};
-        warningInfo['newAssetName'] = newAssetName;
-        warningInfo['oldAssetName'] = oldAssetName;
+        warningInfo["newAssetName"] = newAssetName;
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js:151]: Codacy Issue found: [Avoid using global variables](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5396940)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js [hunk 13]
```diff
@@ -140,284 +148,264 @@ dojo.declare("dotcms.dijit.form.FileAjaxUploader", [dijit._Widget, dijit._Templa
...
+        warningInfo["oldAssetName"] = oldAssetName;
         let message = dojo.replace(this.replaceAssetNameWarning, warningInfo);
 
         let fileInfo = {};
-        fileInfo['newAssetName'] = newAssetName;
-        fileInfo['oldAssetName'] = oldAssetName;
-        fileInfo['message'] = message;
+        fileInfo["newAssetName"] = newAssetName;
+        fileInfo["oldAssetName"] = oldAssetName;
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js:157]: Codacy Issue found: [Avoid using global variables](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5396940)

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js [hunk 14]
```diff
@@ -140,284 +148,264 @@ dojo.declare("dotcms.dijit.form.FileAjaxUploader", [dijit._Widget, dijit._Templa
...
-        warningInfo['oldAssetName'] = oldAssetName;
+        warningInfo["newAssetName"] = newAssetName;
+        warningInfo["oldAssetName"] = oldAssetName;
         let message = dojo.replace(this.replaceAssetNameWarning, warningInfo);
 
         let fileInfo = {};
-        fileInfo['newAssetName'] = newAssetName;
-        fileInfo['oldAssetName'] = oldAssetName;
-        fileInfo['message'] = message;
+        fileInfo["newAssetName"] = newAssetName;
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js:156]: Codacy Issue found: [Avoid using global variables](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5396940)

**Review by jdotcms — APPROVED** [review]

**Review by freddyucv — APPROVED** [review]

**Review by erickgonzalez — APPROVED** [review]

**Review by nollymar — APPROVED** [review]

**Pull request merged**